internal/db/schema/migrations/oss/postgres/44/01_key_rewrap.up.sql

```diff
@@ -0,0 +1,23 @@
+-- make the required schema changes to upgrade the dependency:
```

Comment:
File needs to be moved to migration directory `47`

Reply:
done